admin/src/app/hotel.service.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { PageParameters } from 'src/app/Core/pageParameters';
+
+export interface HotelDto {
+  hotelId: number;
+  name: string;
+  country: string;
+  city: string;
+}
```

Comment:
I'm not sure it's a good place for that code

Reply:
Well, it might be separate file with this interface. I do not see any problems with this interface location, but you do so I will be glad to discuss this point